index.html

```diff
@@ -2399,6 +2399,51 @@ <h4>Not Mapped</h4>
 					<span class="property not-mapped"><a href="#not_mapped">Not mapped*</a></span>
 				</td>
 			</tr>
+			<tr id="ariaBraillelabel">
+				<th><a class="property-reference" href="#aria-braillelabel"><code>aria-braillelabel</code></a></th>
+				<td class="attr-msaa-ia2">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.braillelabel</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleLabel</span>
+				</td>
+			</tr>
+			<tr id="ariaBrailleroledescription">
+				<th><a class="property-reference" href="#aria-brailleroledescription"><code>aria-brailleroledescription</code></a></th>
+				<td class="attr-msaa-ia2">
+					<span class="property">Object Attribute: <code>brailleroledescription:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.brailleroledescription</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>brailleroledescription:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleRoleDescription</span>
+				</td>
+			</tr>
+			<tr id="ariaBrailleroledescriptionUndefined">
+				<th><a class="property-reference" href="#aria-brailleroledescription"><code>aria-brailleroledescription</code></a> is undefined, contains only whitespace characters, or the empty Braille pattern: dots-0 (U+2800)</th>
```

Comment:
```suggestion
				<th><a class="property-reference" href="#aria-brailleroledescription"><code>aria-brailleroledescription</code></a> is undefined the empty string</th>
```

per discussion in PR #135 and [related WG decision](https://www.w3.org/2022/08/11-aria-minutes#t05), should we remove all the references to whitespace here?

Reply:
yes I'll fix now, thanks! :)